ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -161,16 +161,17 @@ public bool HasUpToDateBundleInCache()
             if (_isDevSupportEnabled)
             {
 #if WINDOWS_UWP
-                var lastUpdateTime = Windows.ApplicationModel.Package.Current.InstalledDate.UtcDateTime;
+                var lastNativeUpdateTime = Windows.ApplicationModel.Package.Current.InstalledDate.UtcDateTime;
                 var localFolder = ApplicationData.Current.LocalFolder.Path;
 #else
-                var lastUpdateTime = File.GetCreationTime(Assembly.GetExecutingAssembly().Location);
+                var lastNativeUpdateTime = File.GetLastWriteTimeUtc(Assembly.GetEntryAssembly().Location);
```

Comment:
Assembly.GetExecutingAssembly().Location is going to get the path to the ReactNative.Net46.DLL and not the application's EXE.   Now using Assembly.GetEntryAssembly(), which corrects that.